core/src/main/java/org/apache/accumulo/core/Constants.java

```diff
@@ -42,11 +42,11 @@
   public static final String ZNAMESPACE_NAME = "/name";
   public static final String ZNAMESPACE_CONF = "/conf";
 
-  public static final String ZMASTERS = "/masters";
-  public static final String ZMASTER_LOCK = ZMASTERS + "/lock";
-  public static final String ZMASTER_GOAL_STATE = ZMASTERS + "/goal_state";
-  public static final String ZMASTER_REPLICATION_COORDINATOR_ADDR = ZMASTERS + "/repl_coord_addr";
-  public static final String ZMASTER_TICK = ZMASTERS + "/tick";
+  public static final String ZMANAGERS = "/masters";
```

Comment:
I didn't rename the ZK dir to "managers" because that prevents the manager from starting without initialization. It seemed like too much for a minor release to require a user run some fixup script before launching 2.1.

Reply:
It's possible we could add some upgrade code for this. I'm not sure what would be involved, though. With upgrade code, we can start relying on more constrained upgrade paths (https://accumulo.apache.org/contributor/versioning#LTM)

Reply:
From a ZK upgrade, I believe it's straightforward--do a recursive copy of /masters to /managers and then delete /masters. However, the existing upgrader runs inside the manager which won't start because the accumulo-service script invokes [SetGoalState](https://github.com/apache/accumulo/blob/main/assemble/bin/accumulo-service#L72) first and that hangs. I suppose I could add a check in SetGoalState to do the upgrade if /masters exists and /managers doesn't. Not sure if that makes sense or not.

Reply:
`SetGoalState` isn't a bad idea since I think it has to always run first. It might be worth it to put the Upgrade code in its own separate utility that an Admin has the option to manually call.

Reply:
That sounds good. I pushed a commit that renames the constant and adds a conversion util that is invoked from SetGoalState (but can also be run separately).